.buildkite/pipeline.linux.yml

```diff
@@ -0,0 +1,139 @@
+steps:
+  - label: "build-gnu-x86"
+    commands:
+     - cargo build --release
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  steps:
+  - label: "build-gnu-x86-mmap"
+    commands:
+     - cargo build --release --features=backend-mmap
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  steps:
+  - label: "build-gnu-arm-mmap"
+    commands:
+     - cargo build --release --features=backend-mmap
+    retry:
+      automatic: false
+    agents:
+      platform: arm.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  steps:
+  - label: "build-musl-arm-mmap"
+    commands:
+     - cargo build --release --features=backend-mmap --target aarch64-unknown-linux-musl
+    retry:
+      automatic: false
+    agents:
+      platform: arm.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  - label: "style"
+    command: cargo fmt --all -- --check
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  - label: "unittests-gnu-x86"
+    commands:
+     - cargo test
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          privileged: true
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  - label: "unittests-gnu-x86-mmap"
+    commands:
+     - cargo test --features=backend-mmap
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          privileged: true
```

Comment:
I don't think you need to be privilegded to run these unit tests. Only the unit tests in KVM need privilegded as they need permission to open /dev/kvm and Buildkite does not support passing `--device` to the container.

Reply:
Right, I'll drop this.